### ISSUE TITLE
XMLParsingException in streaming based GetFeature request

### ISSUE THREAD

**Issue by lgoltz** [deegree/deegree3#391]
We got an exception:
```xml
<ows:ExceptionReport xmlns:ows="http://www.opengis.net/ows/1.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.opengis.net/ows/1.1 http://schemas.opengis.net/ows/1.1.0/owsExceptionReport.xsd" version="2.0.0">
  <ows:Exception exceptionCode="NoApplicableCode">
    <ows:ExceptionText>Error in XML document (file 'unknown source', line: 1, column: 6652, character offset: 6651): Required element Reference was not found.</ows:ExceptionText>
  </ows:Exception>
</ows:ExceptionReport>
```
with Stacktrace:
```
[13:35:43] Lyn Goltz: [13:35:32] TRACE: [WebFeatureService] Error in XML document (file 'unknown source', line: 1, column: 462, character offset: 461): Required element Reference was not found.
class org.deegree.commons.xml.XMLParsingException: Error in XML document (file 'unknown source', line: 1, column: 462, character offset: 461): Required element Reference was not found.
<< is empty >>
	at org.deegree.commons.xml.stax.XMLStreamUtils.skipToRequiredElement(XMLStreamUtils.java:524)
	at org.deegree.services.wfs.format.gml.BufferableXMLStreamWriter.appendBufferedXML(BufferableXMLStreamWriter.java:121)
	at org.deegree.services.wfs.format.gml.request.GmlGetFeatureHandler.doGetFeatureResults(GmlGetFeatureHandler.java:290)
	at org.deegree.services.wfs.format.gml.GmlFormat.doGetFeature(GmlFormat.java:238)
	at org.deegree.services.wfs.WebFeatureService.doKVP(WebFeatureService.java:590)
	at org.deegree.services.controller.OGCFrontController.dispatchKVPRequest(OGCFrontController.java:805)
	at org.deegree.services.controller.OGCFrontController.doGet(OGCFrontController.java:369)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:617)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:717)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.deegree.client.core.filter.InputFileFilter.doFilter(InputFileFilter.java:75)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:235)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:470)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:127)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:298)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:857)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:588)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:489)
	at java.lang.Thread.run(Thread.java:744)
```

**Comment by copierrj** [comment]
Could you provide some additional information about how to reproduce this error?

**Comment by lgoltz** [comment]
Fixed by #390.
